methods/incidentReports.coffee

```diff
@@ -35,18 +47,20 @@ Meteor.methods
       Meteor.call("editUserEventLastIncidentDate", incident.userEventId)
     return incident._id
 
-  editIncidentReport: (incident) ->
+  editIncidentReport: (incident, userEventId) ->
```

Comment:
I don't think the userEventId parameter should be added here. I don't see any updated calls to this method, and adding incidents to events would be better to do through the "addIncidentReportToEvent" method.